### DISCUSSION TITLE
redis sentinel for conductor HA

### DISCUSSION THREAD

**Discussion by ritu-p** [Netflix/conductor#2677] [General]
Has anyone configured 3 node HA conductor without using dynomite as persistence? I am seeing some issues such as below when one of the nodes is down.
 java.lang.RuntimeException: com.netflix.dyno.connectionpool.exception.PoolOfflineException: PoolOfflineException: [host=Host [hostname=UNKNOWN, ipAddress=UNKNOWN, port=0, rack: UNKNOWN, datacenter: UNKNOW, status: Down, hashtag=null, redacted latency=0(0), attempts=0]host pool is offline and no Racks available for fallback.

is this expected ?
 Is it a good idea to use redis_sentinel directly in place of dynamo

**Comment by apanicker-nflx** [comment]
@RuriRyan Can you please provide your inputs on running Conductor with redis sentinel configuration? Thanks

**Comment by RuriRyan** [comment]
@ritu-p please have a look at https://github.com/Netflix/conductor/discussions/2321#discussioncomment-947887 where I pasted some sample configuration which we use in our production setup.
Altough we haven't updated conductor in a while, but I kinda doubt much has changed in the regards of persistence. We use v2.31.0. 

And an additional config line for the log4j vuln in this version:
```
      log4j2.formatMsgNoLookups=true
```
